clusters/l3-sqnc/monitoring/kustomization.yaml

```diff
@@ -10,6 +10,7 @@ resources:
   - nginx
   - redis
   - teams-alerts.yaml
+  - deployment-alert.yaml
```

Comment:
Move all the files in this to base and then have it as a part of the flux-system NS.

Reply:
moved, I tested with kind